css/main.scss

```diff
@@ -435,6 +435,37 @@ i:hover {
   }
 }
 
+.cookie-banner {
+  width: 100%;
+  padding: 20px 0;
+  position: fixed;
+  bottom: 0;
+  z-index: 9999;
+  background-color: #e7f2ef;
```

Comment:
Let's make this blue as otherwise it looks like this when scrolling down: 
<img width="1038" alt="bildschirmfoto 2018-01-24 um 14 36 52" src="https://user-images.githubusercontent.com/1510/35334861-0a7320d6-0114-11e8-8018-74aac93900e3.png">